go.mod

```diff
@@ -38,6 +38,7 @@ require (
 	github.com/containerd/typeurl v0.0.0-20190228175220-2a93cfde8c20 // indirect
 	github.com/containernetworking/cni v0.7.1
 	github.com/coredns/corefile-migration v1.0.2
+	github.com/coreos/bbolt v1.3.1-coreos.6 // indirect
 	github.com/coreos/etcd v3.3.15+incompatible
```

Comment:
I expected to see us drop `github.com/coreos/etcd` as a dependency... is there a reason we did not?

Reply:
yes, I am going to update go.mod, haven't pushed it to this PR yet, as it's still WIP. I am still finding my way around updating vendor. Should I resolve building error with updated etcd 3.4.1 first, or should I remove the old github.com/coreos/etcd from go.mod first? After I remove  github.com/coreos/etcd from go.mod, do I need to update-vendor.sh again?